### PR TITLE
[Feature] Menu: add selection mode to menu

### LINKED ISSUE #875: Menu item state management
### Prerequisites

- [x] I have [searched](https://github.com/trimble-oss/modus-wc-2.0/issues?q=is%3Aissue) for duplicate or closed issues
- [x] I have read the [contributing guidelines](https://github.com/trimble-oss/modus-wc-2.0/blob/main/CONTRIBUTING.md)

### Describe the issue

Single-Select Mode (mutually exclusive Active state)

Multi-Select Mode (checkbox-based, multiple selections allowed)

The component must clearly separate Active, Checked, and Focus states and meet accessibility standards.

**Acceptance Criteria:**

**AC1 – Mode Behavior**

No checkboxes → Single-Select Mode.

Checkboxes visible → Multi-Select Mode.

Active and Checked states must not conflict.

**AC2 – Single-Select (Active State)**

Only one item can be Active at a time.

Selecting a new item removes previous Active state.

Click or Enter/Space activates item.

Arrow keys move focus only.

Disabled items are non-interactive.

ARIA: role="menuitemradio" with only one aria-checked="true".

**AC3 – Multi-Select (Checkbox State)**

Multiple items may be Checked simultaneously.

Toggling one item does not affect others.

Space toggles checkbox; arrows move focus.

Disabled items are non-interactive.

ARIA: role="menuitemcheckbox" with correct aria-checked.

**AC4 – Shared Requirements**

Each item must support:

Default

Hover

Focus

Active (single-select only)

Checked (multi-select only)

Disabled

Focus must be visually distinct from Active and Checked.

Edge cases :

If a checked item becomes Disabled → It remains checked but non-interactive.

Removing a checked item must update selection state cleanly.

Focus must not clear checked states.

### Reduced test cases

_No response_

### What operating system(s) are you seeing the problem on?

_No response_

### What browser(s) are you seeing the problem on?

_No response_

### What is the issue regarding?

@trimble-oss/modus-wc

### What version of npm package are you using?

_No response_

### Priority

Medium

### What product/project are you using Modus Web Components for?

Modus

### What is your team/division name?

Modus

### Are you willing to contribute?

None

### Are you using Modus Web Components in production?

_No response_

### Estimate

2 Points

### PR DESCRIPTION
**Pull request by Kavin-Trimble**
### :page_facing_up: Summary of Changes
Menu:
add selection mode to the menu.
change the selected and active states to the same default active state
update the corresponding affected components which uses the selected state

### :thought_balloon: Type of Change

- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)
- [ ] Configuration change (modifies scaling or properties of existing infrastructure & services)
- [ ] Documentation change
- [ ] Other

### :clipboard: Test Plan

[comment]: # 'How do you know the changes are safe to ship to production?'
[comment]: # 'How do you know the changes will function as expected in future releases?'

### :white_check_mark: Self Code Review Checklist

PR authors and reviewers, please verify that all of these items have been completed.

- [ ] My code is clean and readable
- [ ] I followed standard conventions
- [ ] Component code is WCAG 2.2 compliant (if applicable)
- [ ] I have made theme files changes (if applicable)
- [ ] I have made documentation changes (if applicable)
- [ ] I have made Storybook changes including usage documentation (if applicable)
- [ ] I have tested the component thoroughly in Storybook including RTL rendering (if applicable)

### :link: Work Item

Issue #875

### CODE REVIEW

File: src/components/modus-wc-menu-item/modus-wc-menu-item.tsx [hunk 1]
```diff
@@ -137,6 +149,26 @@ export class ModusWcMenuItem {
     return this.hasSubmenu ? 'modus-wc-menu-dropdown-toggle' : '';
   }
 
+  private getSelectionMode(): string | undefined {
+    return (
+      this.el.closest('modus-wc-menu') as HTMLElement & {
+        selectionMode?: string;
+      }
+    )?.selectionMode;
+  }
```
[the remark this answers is not in the document]
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu-item/modus-wc-menu-item.tsx]: addressed

File: src/components/modus-wc-menu/modus-wc-menu.stories.ts [hunk 2]
```diff
@@ -94,6 +100,42 @@ export const Default: Story = {
...
+    // prettier-ignore
+    return html`
+<modus-wc-menu
+  aria-label="Menu"
+  ?bordered=${args.bordered}
+  custom-class=${ifDefined(args['custom-class'])}
+  orientation=${ifDefined(args.orientation)}
+  selection-mode=${ifDefined(args['selection-mode'])}
+  size=${ifDefined(args.size)}
+>
```
[the remark this answers is not in the document]
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu/modus-wc-menu.stories.ts:133]: updated

File: src/components/modus-wc-menu/modus-wc-menu.stories.ts [hunk 3]
```diff
@@ -200,6 +242,7 @@ export const CollapsibleMenu: Story = {
         <modus-wc-menu-item
           label="Charts"
           .hasSubmenu=${true}
+          checkbox="true"
```
[the remark this answers is not in the document]
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu/modus-wc-menu.stories.ts]: removed

File: src/components/modus-wc-menu-item/modus-wc-menu-item.tsx [hunk 4]
```diff
@@ -137,6 +149,26 @@ export class ModusWcMenuItem {
...
+
+  private deselectSiblings(): void {
+    const menu = this.el.closest('modus-wc-menu');
+    if (!menu) return;
+
+    const siblings = menu.querySelectorAll('modus-wc-menu-item');
+    siblings.forEach((item) => {
```
[the remark this answers is not in the document]
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu-item/modus-wc-menu-item.tsx]: updated

File: src/components/modus-wc-menu-item/modus-wc-menu-item.spec.ts [hunk 5]
```diff
@@ -590,4 +591,226 @@ describe('modus-wc-menu-item', () => {
...
+    const menuItem = page.doc.querySelector(
+      'modus-wc-menu-item'
+    ) as HTMLElement;
+    const button = menuItem.querySelector('button') as HTMLButtonElement;
+
+    // Click to check
+    button.click();
+    await page.waitForChanges();
+
+    const liElement = menuItem.querySelector('li') as HTMLLIElement;
```
[the remark this answers is not in the document]
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu-item/modus-wc-menu-item.spec.ts:646]: updated

File: src/components/modus-wc-menu/modus-wc-menu.spec.ts [hunk 6]
```diff
@@ -73,6 +73,335 @@ describe('modus-wc-menu', () => {
...
+      html: `
+        <modus-wc-menu aria-label="Test menu">
+          <modus-wc-menu-item label="Item 1" value="1"></modus-wc-menu-item>
+          <modus-wc-menu-item label="Item 2" value="2"></modus-wc-menu-item>
+        </modus-wc-menu>
+      `,
+    });
+
+    const menuItems = page.doc.querySelectorAll('modus-wc-menu-item');
+    const firstLi = menuItems[0].querySelector('li') as HTMLLIElement;
```
[the remark this answers is not in the document]
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu/modus-wc-menu.spec.ts]: updated

File: src/components/modus-wc-menu/modus-wc-menu.stories.ts [hunk 7]
```diff
@@ -94,6 +100,42 @@ export const Default: Story = {
   },
 };
 
+export const MultiSelectMenu: Story = {
+  render: (args) => {
+    args['selection-mode'] = 'multiple';
```
[the remark this answers is not in the document]
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu/modus-wc-menu.stories.ts]: updated

File: src/components/modus-wc-menu-item/modus-wc-menu-item.tsx [hunk 8]
```diff
@@ -232,12 +273,12 @@ export class ModusWcMenuItem {
...
             <div class="modus-wc-menu-item-content">
-              {this.checkbox && (
+              {(this.checkbox || this.isMultiSelect()) && (
                 <modus-wc-checkbox
                   aria-label="Checkbox"
                   disabled={this.disabled}
                   size={this.size}
-                  value={!!this.selected}
+                  value={this.isMultiSelect() ? this.checked : !!this.selected}
                 />
```
[the remark this answers is not in the document]
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu-item/modus-wc-menu-item.tsx:304]: updated

File: src/components/modus-wc-menu/modus-wc-menu.tsx [hunk 9]
```diff
@@ -37,6 +37,9 @@ export class ModusWcMenu {
   /** The orientation of the menu. */
   @Prop() orientation?: Orientation = 'vertical';
 
+  /** The selection mode of the menu. */
+  @Prop() selectionMode?: SelectionMode = 'single';
```
Comment (ElishaSamPeterPrabhu) [inline src/components/modus-wc-menu/modus-wc-menu.tsx]: `modus-wc-menu-item` reads the parent menu's `selectionMode` via DOM traversal (`this.el.closest('modus-wc-menu')?.selectionMode`). This works as long as `selectionMode` is accessed as a JS property. However, since `modus-wc-menu` uses `shadow: false`, slotted children render in the light DOM and the `closest()` call will work — but only on first render. If `selectionMode` changes at runtime (e.g., a consumer switches from `'single'` to `'multiple'`), the children will **not re-render** because there is no `@Watch` or prop forwarding triggering re-renders on menu items. This is a reactivity gap that needs addressing.

File: src/components/modus-wc-menu/modus-wc-menu.tsx [hunk 10]
```diff
@@ -56,6 +61,17 @@ export class ModusWcMenu {
...
+  componentDidLoad() {
+    this.ulElement = this.el.querySelector('ul') as HTMLUListElement;
+    this.ulElement?.addEventListener('keydown', this.handleKeyDown);
+    this.ulElement?.addEventListener('focusout', this.handleFocusout);
+  }
+
+  disconnectedCallback() {
+    this.ulElement?.removeEventListener('keydown', this.handleKeyDown);
+    this.ulElement?.removeEventListener('focusout', this.handleFocusout);
+  }
```
Comment (ElishaSamPeterPrabhu) [inline src/components/modus-wc-menu/modus-wc-menu.tsx]: `componentDidLoad` caches `this.ulElement` once. Stencil may re-render and re-create the `<ul>` element, leaving `ulElement` pointing to a detached DOM node. The keyboard listener would then be on a ghost element. The Stencil-native approach (using JSX event binding or `@Listen`) would avoid this entirely.

File: src/components/modus-wc-menu-item/modus-wc-menu-item.tsx [hunk 11]
```diff
@@ -137,6 +159,40 @@ export class ModusWcMenuItem {
     return this.hasSubmenu ? 'modus-wc-menu-dropdown-toggle' : '';
   }
 
+  private getSelectionMode(): 'single' | 'multiple' | undefined {
```
Comment (ElishaSamPeterPrabhu) [inline src/components/modus-wc-menu-item/modus-wc-menu-item.tsx]: The `SelectionMode` type already exists in `../types`. This should use it:
```ts
import { ModusSize, SelectionMode } from '../types';
// ...
private getSelectionMode(): SelectionMode | undefined {
  return (
    this.el.closest('modus-wc-menu') as HTMLElement & {
      selectionMode?: SelectionMode;
    }
  )?.selectionMode;
}
```
The inline union at lines 164–165 for the cast also needs updating.

Comment (ElishaSamPeterPrabhu) [inline src/components/modus-wc-menu-item/modus-wc-menu-item.tsx]: It's called from `isMultiSelect()`, `getRole()`, `getAriaChecked()`, `getClasses()` (via `isMultiSelect()`), and `handleItemSelect()`. Each call does `closest('modus-wc-menu')`. It should be computed once:

```ts
private getClasses(): string {
  const mode = this.getSelectionMode(); // once
  const isMulti = mode === 'multiple';
  // pass mode/isMulti down to getRole, getAriaChecked, etc.
}
```

File: src/components/modus-wc-menu-item/modus-wc-menu-item.tsx [hunk 12]
```diff
@@ -137,6 +159,40 @@ export class ModusWcMenuItem {
...
+    const menu = this.el.closest('modus-wc-menu');
+    if (!menu) return;
+
+    const siblings = menu.querySelectorAll('modus-wc-menu-item');
+    siblings.forEach((item) => {
+      if (item !== this.el && item.closest('modus-wc-menu') === menu) {
+        (item as HTMLElement & { selected?: boolean }).selected = false;
+      }
+    });
+  }
```
Comment (ElishaSamPeterPrabhu) [inline src/components/modus-wc-menu-item/modus-wc-menu-item.tsx]:
```ts
(item as HTMLElement & { selected?: boolean }).selected = false;
```
Setting `.selected` directly on a sibling element sets the **DOM property**, not the Stencil `@Prop`. In Stencil with `shadow: false`, props can be reflected as attributes, but direct property mutation on another component instance does **not** trigger that instance's change detection or re-render. The sibling's `aria-checked` will not update, making the ARIA state stale. The correct fix is either:
- Use `item.setAttribute('selected', 'false')` to reflect via attribute (if the prop uses `reflect: true`)
- Or emit a `deselectAll` event on the parent menu and handle it there

File: src/components/modus-wc-menu-item/modus-wc-menu-item.tsx [hunk 13]
```diff
@@ -75,20 +75,36 @@ export class ModusWcMenuItem {
...
     // Auto-inject CSS if component is used inside user's shadow DOM
     handleShadowDOMStyles(this.el);
 
     this.inheritedAttributes = inheritAriaAttributes(this.el);
   }
 
+  componentDidLoad() {
+    this.liElement = this.el.querySelector('li') as HTMLLIElement;
+    this.liElement?.addEventListener('keydown', this.handleKeyDown);
+  }
```
Comment (ElishaSamPeterPrabhu) [inline src/components/modus-wc-menu-item/modus-wc-menu-item.tsx]: The previous approach used `onKeyDown={this.handleKeyDown}` directly on the `<li>` in JSX. The new approach attaches it imperatively in `componentDidLoad`. This is inconsistent with the rest of the component (e.g. `onClick={this.handleItemSelect}` is still JSX-bound on the `<button>`). Worse, if Stencil re-renders and recreates the `<li>`, the cached `this.liElement` reference becomes stale and the listener is lost. Keep event handling in JSX or use Stencil's `@Listen` decorator consistently.

File: src/components/modus-wc-menu/modus-wc-menu.stories.ts [hunk 14]
```diff
@@ -94,6 +100,45 @@ export const Default: Story = {
...
+  <modus-wc-menu-item
+    label="Menu Item 4"
+    value="3"
+    bordered="true"
+  ></modus-wc-menu-item>
+  <modus-wc-menu-item
+    label="Menu Item 5"
+    value="3"
+    sub-label="Menu Item 5 Sub-label"
+  ></modus-wc-menu-item>
```
Comment (ElishaSamPeterPrabhu) [inline src/components/modus-wc-menu/modus-wc-menu.stories.ts:152]: Items 4 and 5 should have `value="4"` and `value="5"`. Values must be unique — they are what gets emitted in the `itemSelect` event and are how consumers identify which items were selected.

File: src/components/modus-wc-menu/modus-wc-menu.stories.ts
Comment (ElishaSamPeterPrabhu) [inline src/components/modus-wc-menu/modus-wc-menu.stories.ts:24]: The `selection-mode` control is added to `argTypes` but there is no default value set in `args`. This means the control starts blank and consumers won't know what the default is. Since the prop defaults to `'single'`, the story args should reflect that.

**Review by ElishaSamPeterPrabhu — COMMENTED** [review]
The default `selectionMode="single"` causes two breaking changes:

**1. Behavioral breaking change** — The existing menu had no enforced selection constraint. Consumers could have multiple items with `selected="true"` at the same time and the component didn't interfere. With `selectionMode="single"` as the default, clicking any item now calls `deselectSiblings()` which removes `selected` from all other items.

**2. ARIA breaking change** — The existing menu used `role="menuitem"` with no selection ARIA attributes at all. With `selectionMode="single"` as the default, every existing menu item now becomes `role="menuitemradio"` with `aria-checked`.

The side nav story has:
```html
<modus-wc-menu size="lg">
  <modus-wc-menu-item label="home" selected></modus-wc-menu-item>
  <modus-wc-menu-item label="profile"></modus-wc-menu-item>
  <modus-wc-menu-item label="settings"></modus-wc-menu-item>
</modus-wc-menu>
```
Screen reader announces: *"home, menu item"* / *"profile, menu item"* / *"settings, menu item"*

`selectionMode` now defaults to `'single'`:

Screen reader announces: *"home, radio button, checked, 1 of 3"* / *"profile, radio button, not checked, 2 of 3"* / *"settings, radio button, not checked, 3 of 3"*

The menu is a navigation sidebar, not a radio group. But every screen reader user now hears "radio button" for every nav item, implying a mutually exclusive selection choice rather than navigation.

Defaulting to `'multiple'` would not fix either problem, it would just replace one wrong default with a different one, injecting an unexpected checkbox into every existing menu item and announcing `"home, check box, checked"` instead.

**The only correct fix is `none` as the default.** That's the only value that:
1. Keeps `role="menuitem"` — preserving the original ARIA announcement
2. Skips `deselectSiblings()` — consumers managing `selected` on multiple items externally won't have that state silently reset on click
3. Skips the auto-injected checkbox — preserving the visual appearance

@Sunder005 @Kavin-Trimble

**Comment by ElishaSamPeterPrabhu** [comment]
> The default `selectionMode="single"` causes two breaking changes:
> 
> **1. Behavioral breaking change** — The existing menu had no enforced selection constraint. Consumers could have multiple items with `selected="true"` at the same time and the component didn't interfere. With `selectionMode="single"` as the default, clicking any item now calls `deselectSiblings()` which removes `selected` from all other items.
> 
> **2. ARIA breaking change** — The existing menu used `role="menuitem"` with no selection ARIA attributes at all. With `selectionMode="single"` as the default, every existing menu item now becomes `role="menuitemradio"` with `aria-checked`.
> 
> The side nav story has:
> 
> ```
> <modus-wc-menu size="lg">
>   <modus-wc-menu-item label="home" selected></modus-wc-menu-item>
>   <modus-wc-menu-item label="profile"></modus-wc-menu-item>
>   <modus-wc-menu-item label="settings"></modus-wc-menu-item>
> </modus-wc-menu>
> ```
> 
> Screen reader announces: _"home, menu item"_ / _"profile, menu item"_ / _"settings, menu item"_
> 
> `selectionMode` now defaults to `'single'`:
> 
> Screen reader announces: _"home, radio button, checked, 1 of 3"_ / _"profile, radio button, not checked, 2 of 3"_ / _"settings, radio button, not checked, 3 of 3"_
> 
> The menu is a navigation sidebar, not a radio group. But every screen reader user now hears "radio button" for every nav item, implying a mutually exclusive selection choice rather than navigation.
> 
> Defaulting to `'multiple'` would not fix either problem, it would just replace one wrong default with a different one, injecting an unexpected checkbox into every existing menu item and announcing `"home, check box, checked"` instead.
> 
> **The only correct fix is `none` as the default.** That's the only value that:
> 
> 1. Keeps `role="menuitem"` — preserving the original ARIA announcement
> 2. Skips `deselectSiblings()` — consumers managing `selected` on multiple items externally won't have that state silently reset on click
> 3. Skips the auto-injected checkbox — preserving the visual appearance
> 
> @Sunder005 @Kavin-Trimble


After our discussion this is the approach we are going forward with:

- **`selectionMode="single"`:** `role="menuitemradio"` + `aria-checked` — correct ARIA for radio-style single selection
- **`selectionMode="multiple"`:** `role="menuitemcheckbox"` + `aria-checked` — correct ARIA for multi-select

File: src/components/modus-wc-menu/modus-wc-menu.stories.ts
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu/modus-wc-menu.stories.ts:24]: addressed

File: src/components/modus-wc-menu/modus-wc-menu.stories.ts (hunk 14 above)
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu/modus-wc-menu.stories.ts:152]: Addressed

File: src/components/modus-wc-menu-item/modus-wc-menu-item.tsx (hunk 13 above)
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu-item/modus-wc-menu-item.tsx]: Addressed

File: src/components/modus-wc-menu-item/modus-wc-menu-item.tsx (hunk 11 above)
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu-item/modus-wc-menu-item.tsx]: Addressed

Comment (Kavin-Trimble) [reply src/components/modus-wc-menu-item/modus-wc-menu-item.tsx]: Addressed

File: src/components/modus-wc-menu/modus-wc-menu.tsx (hunk 10 above)
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu/modus-wc-menu.tsx]: Addressed

File: src/components/modus-wc-menu/modus-wc-menu.tsx (hunk 9 above)
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu/modus-wc-menu.tsx]: Addressed

File: src/components/modus-wc-menu-item/modus-wc-menu-item.tsx (hunk 12 above)
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu-item/modus-wc-menu-item.tsx]: Addressed

**Comment by Kavin-Trimble** [comment]
moving to review @ElishaSamPeterPrabhu

File: src/components/modus-wc-menu-item/modus-wc-menu-item.tsx [hunk 15]
```diff
@@ -72,25 +74,54 @@ export class ModusWcMenuItem {
...
+      this.parentMenuObserver.observe(parentMenu, { attributes: true });
+    }
+  }
+
+  disconnectedCallback() {
+    this.parentMenuObserver?.disconnect();
+  }
+
+  private handleSelectionModeChange(): void {
+    this.selected = this.selected === undefined ? false : undefined;
```
Comment (ElishaSamPeterPrabhu) [inline src/components/modus-wc-menu-item/modus-wc-menu-item.tsx]: This flips `selected` to false if it was `undefined`, and `undefined` if it had a value. The intent is to reset state when selectionMode changes, but the logic toggles incorrectly. It should always reset to `undefined` (or false)
Because of this, when `selectionMode` is undefined in Storybook, I can select multiple items.

Image

File: src/components/modus-wc-menu-item/modus-wc-menu-item.tsx [hunk 16]
```diff
@@ -112,18 +143,41 @@ export class ModusWcMenuItem {
...
+    const menu = this.el.closest('modus-wc-menu');
+    if (!menu) return;
+
+    const siblings = menu.querySelectorAll('modus-wc-menu-item');
+    siblings.forEach((item) => {
+      if (item !== this.el && item.closest('modus-wc-menu') === menu) {
+        (item as HTMLElement & { selected?: boolean }).selected = false;
+      }
+    });
+  }
```
Comment (ElishaSamPeterPrabhu) [inline src/components/modus-wc-menu-item/modus-wc-menu-item.tsx:169]: Since we’re handling it this way, single-item selection might not be captured when the lifecycle is managed by another renderer like React, unlike Stencil here. Can you verify this in React?

**Review by ElishaSamPeterPrabhu — CHANGES_REQUESTED** [review]

**Review by ElishaSamPeterPrabhu — CHANGES_REQUESTED** [review]
Multiple Items should be emitted via an array

File: src/components/modus-wc-menu-item/modus-wc-menu-item.tsx (hunk 15 above)
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu-item/modus-wc-menu-item.tsx]: Modified single selection as default

**Comment by Kavin-Trimble** [comment]
> Multiple Items should be emitted via an array

addressed

File: src/components/modus-wc-menu-item/modus-wc-menu-item.tsx (hunk 16 above)
Comment (Kavin-Trimble) [reply src/components/modus-wc-menu-item/modus-wc-menu-item.tsx:169]: https://github.com/user-attachments/assets/81f39c63-e060-4fc9-9012-f9b4e2d107fc

Here is what i have tested with react. The selection mode is getting updated in the runtime. Could you verify it @ElishaSamPeterPrabhu

**Review by ElishaSamPeterPrabhu — DISMISSED** [review]
Moving to QA @jewel-shajan

**Comment by jewel-shajan** [comment]
@Kavin-Trimble Here are few observations after testing:

- Collapsible menus or multi-level menu selection scenarios are not currently handled. As a result, the side navigation and  panel is impacted.
- In single-selection mode, the aria-checked attribute is present—please confirm if this is intentional.
- Do the Active, Checked, and Focus states each require corresponding ARIA attributes?
- When the menu item state is changed to Active in the DOM, it does not align with the Selected state. Is this expected behavior?
- In multi-selection mode, the text color does not appear correct.

**Review by ElishaSamPeterPrabhu — APPROVED** [review]
Moving to QA @jewel-shajan

**Comment by jewel-shajan** [comment]
As per discussion with the team
- Currently we are not addressing multiple selection in multi menu scenarios.
- In focused state there wont be aria attribute.
@Kavin-Trimble testing completed and ready to merge.
